### DISCUSSION TITLE
Book view mode

### DISCUSSION THREAD

**Discussion by tahiraj07** [stephanrauh/ngx-extended-pdf-viewer#1126] [Q&A]
When you hover mouse on the first page of the pdf viewer it appears the second page without clicking on it. How can this be handled?

**Comment by stephanrauh** [comment]
Well, that's not what I see on https://pdfviewer.net/extended-pdf-viewer/book-mode. You're right, hovering on a page does have some effect. The fist page - "the hard cover" - jumps slightly to the front. Every other page shows a dog's ear. That may be unexpected because you don't need a click, but it doesn't match your description.

In other words, I'm afraid I don't know how to show the second page without clicking. Would you mind to describe what precisely you're doing? A short screencast might help. 

Thanks in advance!
Stephan

**Reply by tahiraj07** [reply]
@stephanrauh  i just want to not show the second page when i hover mouse at corners of pdf(not when i click on corners when i hover the mouse, i am ok with clicking).Where can i change this hover effect?
![firstpage-hover](https://user-images.githubusercontent.com/45981191/150506308-c98c5fe5-70b5-4bf6-adfe-cb20103ec887.png)
![firstpage](https://user-images.githubusercontent.com/45981191/150506323-0204386f-9224-48ef-b2de-d2b02d767b7e.PNG)

**Reply by stephanrauh** [reply]
I know I'm going to sound stupid if I repeat your message in my own words. But it's really confusing because my demo application behaves differently. You're moving the mouse over - say - the upper right corner of the page. You don't click, just hover. And that's enough to display the next page? Adding insult to injury, judging from your screenshot, the next page is displayed mirrored?

When I do the same, it looks like this (on page 3+):

![image](https://user-images.githubusercontent.com/3045767/150595588-216c87cf-2833-4058-a7c2-ab0756866652.png)

The front page is displayed live a hard-cover book, so instead of showing the dog's ear, the page is lifted a little. But it doesn't flip.

**Reply by stephanrauh** [reply]
Just in case you're read the first version of my answer before I edited it: of course my demo application doesn't behave _difficult_. "Differently" was the word I was looking for :)
